### PR TITLE
[5.2] Added view_or_fail helper

### PR DESCRIPTION
**Pull request by caneco**
I don't know that if this is the most correct approach but you have the general idea.

### CODE REVIEW

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -779,3 +779,32 @@ function view($view = null, $data = [], $mergeData = [])
...
     }
 }
+
+
+
+if (! function_exists('viewOrFail')) {
+    /**
+     * Get the evaluated view contents for the given view or throw an exception.
+     *
+     * @param  string  $view
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: string|null

File: src/Illuminate/Foundation/helpers.php [hunk 2]
```diff
@@ -779,3 +779,32 @@ function view($view = null, $data = [], $mergeData = [])
         return $factory->make($view, $data, $mergeData);
     }
 }
+
+
+
+if (! function_exists('viewOrFail')) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: snake_case required

File: src/Illuminate/Foundation/helpers.php [hunk 3]
```diff
@@ -779,3 +779,32 @@ function view($view = null, $data = [], $mergeData = [])
         return $factory->make($view, $data, $mergeData);
     }
 }
+
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: too many new lines

**Comment by taylorotwell** [comment]
No plans to add this to core.

**Comment by caneco** [comment]
The zonda in this one is weak. Still, thanks…

+1 @GrahamCampbell

**Pull request closed without merging**